libs/perun/services/src/lib/ApiInterceptor.ts

```diff
@@ -61,6 +63,7 @@ export class ApiInterceptor implements HttpInterceptor {
         sessionStorage.setItem('auth:redirect', location.pathname);
         sessionStorage.setItem('auth:queryParams', location.search.substring(1));
         void this.initAuthService.handleAuthStart();
+        this.reauthenticate();
```

Comment:
There was a recent change to this and I believe you can remove the hanldeAuthStart but place it inside of the reauthenticate instead of startAuth.